### ISSUE TITLE
Ec2 get 

### ISSUE THREAD

**Issue by riteshsonawane1372** [surajincloud/awsctl#6]
A function ``ec2FunDescribe`` which get info about ec2 instance using the ``GetEc2func``

**Comment by surajnarwade** [comment]
can you explain the command, usage of command in details?

**Comment by riteshsonawane1372** [comment]
``awsctl get ec2 ``

* This will get information about the instance Using the <a href = "https://github.com/riteshsonawane1372/awsctl/blob/main/cmd/ec2.go#L35-L40"> GetEc2Info</a>
which will check if we got the permission and returns a ``DryRunOpertion`` as a response. 

* GetEc2Info will return the information which is in ec2.DescribeInstancesOutput.Reservation type <a href ="https://github.com/riteshsonawane1372/awsctl/blob/main/cmd/ec2.go#L53">check</a> <a href = "https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-instances.html">doc</a>

* Based on the data retrieved we will <a href ="https://github.com/riteshsonawane1372/awsctl/blob/main/cmd/ec2.go#L58-L65">display</a> Reservation ID owner ID Requester ID (Bcoz we are not using root ac) Instance launch time and its ID

**Comment by riteshsonawane1372** [comment]
@surajnarwade I tested it: Created an Ec2 instance and to verify I ran `` awsctl get ec2`` got the ReservationID and some error.
![image](https://user-images.githubusercontent.com/72307121/221210278-54bfd298-1d5f-4b71-9b23-53e5bc1b7573.png) 

I also tested when no instance is running got no output nor error. Will try to fix this error and pls let me know what is required for the ``get cmd``

**Comment by riteshsonawane1372** [comment]
For this I added access key and secret in ~/.aws/credentials 
```
[default]
aws_access_key_id=
aws_secret_access_key=
```
And in ~/.aws/config 
```
[default]
region=
output=
```

**Comment by riteshsonawane1372** [comment]
@surajnarwade  I have fixed the error the pointer was returning nil. Everything is working good. I ran multiple instance to check, pls have a look.

**Comment by surajnarwade** [comment]
this has been solved in #7
